src/constants/plugins/GuiPlugins.ts

```diff
@@ -15,46 +15,6 @@ export const guiPlugins: { [pluginId: string]: GuiPlugin } = {
     originWhitelist: ['https://libertyx.com'],
     permissions: ['location']
   },
-  moonpay: {
```

Comment:
I take it this "moonpay" plugin is unused, and has been ever since we added the explicit credit card flow?

Reply:
Yes, not used anymore although added back to the sell side